README.md

```diff
@@ -1,207 +1,339 @@
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| --agree-terms    | installs LUKSO client and agrees with Terms & Conditions                                                                    |
+| --geth -\*       | \* Pass any flag to the Geth node [See docs for details](https://geth.ethereum.org/docs/fundamentals/command-line-options)  |
+| --erigon -\*     | \* Pass any flag to the Erigon node [See docs for details](https://github.com/ledgerwatch/erigon)                           |
+| --prysm -\*      | \* Pass any flag to the Prysm node [See docs for details](https://docs.prylabs.network/docs/prysm-usage/parameters)         |
+| --lighthouse -\* | \* Pass any flag to the Lighthouse node [See docs for details](https://lighthouse-book.sigmaprime.io/advanced-datadir.html) |
+
+#### Examples
+
+```bash
+# downloads a specific version of (geth/prysm/erigon/lighthouse) client - Example Geth v1.11.4
+$ lukso install --geth-tag
+# downloads a specific tagged commit of (geth/prysm/erigon/lighthouse) client- Example Geth v1.11.4
+$ lukso install --geth-commit-hash
+#  downloads a specific version of the validator
+$ lukso install --validator-tag
+```
+
+## log
+
+Log displays the logs of LUKSO's execution/consensus/validator clients. Here are the common flags:
+
+```bash
+# displays the logs of LUKSO's Execution client
+$ lukso log execution
+# displays the LUKSO's consensus client's logs
+$ lukso log consensus
+# displays the LUKSO's validator client's logs
+$ lukso log validator
+```
+
+| Flag             | Description                                                                                                                 |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| --testnet        | displays client's testnet logs                                                                                              |
+| --devnet         | displays client's devnet logs                                                                                               |
+| --log-folder     | user can access their custom log folder "./myCustomLogFolder" [learn how to setup log folder]()                             |
+| --geth -\*       | \* Pass any flag to the Geth node [See docs for details](https://geth.ethereum.org/docs/fundamentals/command-line-options)  |
+| --erigon -\*     | \* Pass any flag to the Erigon node [See docs for details](https://github.com/ledgerwatch/erigon)                           |
+| --prysm -\*      | \* Pass any flag to the Prysm node [See docs for details](https://docs.prylabs.network/docs/prysm-usage/parameters)         |
+| --lighthouse -\* | \* Pass any flag to the Lighthouse node [See docs for details](https://lighthouse-book.sigmaprime.io/advanced-datadir.html) |
```

Comment:
For logs we don't actually need any of those client-specific flags - all we need is to choose a network and optionally a log-folder